cmake/scripts/expand_instantiations.cc

```diff
@@ -439,7 +439,18 @@ process_instantiations()
   //   for (X:Y; A:B) { INST }
   while (whole_file.size() != 0)
     {
+      // skip space, tabs, comments:
       skip_space(whole_file);
+
+      // output preprocessor defines as is:
+      if (has_prefix(whole_file, "#"))
+      {
+        std::cout << get_substring_with_delim(whole_file, "\n") << '\n';
+        skip_space(whole_file);
+       continue;
```

Comment:
This still looks off. :slightly_smiling_face: Maybe just run `clang-format` over it?

Reply:
Sorry. Fixed. Let me do clang-format in a separate PR.

Reply:
Oh, I was not requesting that we need to run `clang-format` on this file automatically. I tried manually and this was the only change I was seeing.